app/_src/gateway/production/monitoring/health-check.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Health Check
+content_type: tutorial
+---
+
+This tutorial will guide you through the process of using the new Kong Status endpoint, which provides a more reliable and simple way to determine if Kong is ready to serve requests.
+
+To put it simply, health check endpoint will return a `200 OK` response when Kong is ready, or a `503 Service Unavailable` response when it's not. This is useful for load balancers and other tools that need to monitor the readiness of Kong instances.
+
+## Prerequisites
+
+* Kong installed
+* Basic understanding of Kong configuration and deployment modes (Traditional, DB-less, and Hybrid)
+
+## Understanding the Status Endpoint
+
+Before diving into the steps, it's important to understand the purpose of the Status Endpoint and how it determines whether a Kong instance is ready or not.
+
+### Traditional mode
+
+In Traditional mode, the conditions that must be fulfilled simultaneously for the traditional mode are: 
+
+1. Successful connection to the database
+2. Successful initial build of all routers for all workers
+3. Successful initial build of all plugin iterators for all workers
+
+### DB-less mode (data_plane role)
+
+In DB-less mode, the endpoint returns 200 OK when all of the following conditions are met:
+
+1. The current configuration hash is not nil and not all zeros
+2. Successful initial build/rebuild of all routers for all workers
+3. Successful initial build/rebuild of all plugin iterators for all workers
+
+## Configuring the Status Endpoint
+
+To configure the Status Endpoint in your Kong instance, follow these steps:
+
+1. Ensure that Kong is setup.
+2. Open the Kong configuration file (e.g., `kong.conf` or `kong.yml`).
+3. Make sure the Status API is enabled by including the following line:
+
+    ```conf
+    status_listen = 0.0.0.0:8100
+    ```
+
+    This will make the Status API listen on port 8100. You can change the port number to any available port on your system.
```

Comment:
```suggestion
    This will make the Status API server listen on port `8100` on all interfaces of the system. You can change the port number to any available port on your system.
```